README.md

```diff
@@ -25,6 +25,13 @@ This is a shortcut for [`autograder.cli.courses.assignments.submissions.submit`]
 python3 -m autograder.run.submit my_file.py
 ```
 
+To submit an assignment late, use the following command. For more information and examples,
```

Comment:
Break the line on the sentence.
(The break at the clause (comma in this case) is fine.)